marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/4/transforms/mlcp-flow-transform.sjs

```diff
@@ -49,7 +49,7 @@ function transform(content, context) {
   let flow = flowlib.getFlow(entityName, flowName, consts.INPUT_FLOW);
 
   if (!flow) {
-    fn.error(null, "RESTAPI-SRVEXERR", "The specified flow " + params.flow + " is missing.");
+    fn.error(null, "RESTAPI-SRVEXERR", Sequence.from([404, "Not Found", "The specified flow " + params.flow + " is missing."]));
```

Comment:
Let's convert this to use ds-util.sjs too. That library of course is a little mis-named now - we may want to rename it to "http-util.sjs", since the functions are generic to HTTP. But the more error handling that's routed through those functions, the better, as it makes it easier for us to track where we're throwing errors. And of course, it reduces duplication of the usage of RESTAPI-SRVEXERR.

Reply:
The uri of http-utils.sjs will be /data-hub/5/data-services/http-utils.sjs . Should that be changed to /data-hub/5/impl/http-utils.sjs or /data-hub/5/http-utils.sjs ?

Reply:
Oh I was saying - don't rename it now, let's take care of that in a separate PR. Sorry for the confusion.

Reply:
Since we are using it outside the library outside of ds, is it ok to rename in this PR itself ?

Reply:
Sure, that's fine. I'm not a fan of the "impl" directory (since "impl" is meaningless), but putting it in there is consistent with "hub-utils" and "flow-utils", so that'll work for now.

Reply:
I think the PR is ready for review then